executor/index_merge_reader.go

```diff
@@ -397,10 +443,14 @@ func (e *IndexMergeReaderExecutor) startIndexMergeTableScanWorker(ctx context.Co
 		go func() {
 			defer trace.StartRegion(ctx, "IndexMergeTableScanWorker").End()
 			var task *lookupTableTask
+			startTime := time.Now()
 			util.WithRecovery(
 				func() { task = worker.pickAndExecTask(ctx1) },
 				worker.handlePickAndExecTaskPanic(ctx1, task),
 			)
+			if e.stats != nil {
+				atomic.AddInt64(&e.stats.TableRowScan, int64(time.Since(startTime)))
```

Comment:
This time stats are not accurate. Maybe use `wait` and `fetch_rows` time duration in `pickAndExecTask` function?